src/main/java/com/celestabank/celestabankapi/controller/AccountController.java

```diff
@@ -0,0 +1,151 @@
+package com.celestabank.celestabankapi.controller;
+
+import com.celestabank.celestabankapi.entity.Account;
+import com.celestabank.celestabankapi.entity.CurrentAccount;
+import com.celestabank.celestabankapi.entity.SavingAccount;
+import com.celestabank.celestabankapi.entity.Transaction;
+import com.celestabank.celestabankapi.exeption.BalanceNotSufficientException;
+import com.celestabank.celestabankapi.exeption.BankAccountNotFoundException;
+import com.celestabank.celestabankapi.exeption.InvalidAccountException;
+import com.celestabank.celestabankapi.exeption.InvalidDetailsException;
+import com.celestabank.celestabankapi.service.AccountServiceImp;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/account")
+@AllArgsConstructor
+public class AccountController {
+
+    AccountServiceImp   accountServiceImp;
+
+    @PostMapping("/add")
+    public List<Account> addAccount(@RequestBody Account account){
+        return accountServiceImp.addAccount(account);
+    }
+
+    @PostMapping("/savings")
+    public List<SavingAccount> addSavingAcc(@RequestBody SavingAccount savingAccount) throws InvalidDetailsException {
```

Comment:
Spring recommends to use RuntimeExeptions

Reply:
i have some problems to use throws exception to my client